### ISSUE TITLE
The built route is deeply wrong

### ISSUE THREAD

**Issue by Gravity2106** [organicmaps/organicmaps#8121]
⚠ Have you searched for similar, already existing issues? Yes

**Describe the issue**
Please write a clear and concise description of the issue here.
If I tried to go from A to B, the path recommended is very bad in comparison with the best one possible and the stimated time is not reliable at all (this last one maybe due to traffic lights?).
OSM does it well.

**Steps to reproduce**

**Expected behaviour**
The app recommends the best route available as OSM does.

**Screenshots**
If applicable, add screenshots or screen recordings to help explain your problem.

**System information:**
 - Android 14
 - versión 2024.03.31-8-Web
app.organicmaps.web
versionCode 24033108
targetSdk 34
minSdk 21
Installed: 4 abr 2024; 16:49
 - Pixel 7

**Additional context**
Please add any other context or comments here that may be useful.

**Comment by Jean-BaptisteC** [comment]
Without start and end point of your routing, you can't doing nothing :)

**Comment by Jean-BaptisteC** [comment]
Is it possible to give approximative coordinates or just part of wrong routing?

**Comment by Jean-BaptisteC** [comment]
Send an email to hello@organicmaps.app

**Comment by biodranik** [comment]
Results can be different (esp. estimated time), because different engines calculate it differently. Organic Maps doesn't take into account turns and traffic lights yet, for example.

Without knowing the coordinates of a route, we can't improve it. You can share them privately with screenshots at hello or support [at] organicmaps dot app

**Comment by matheusgomesms** [comment]
Yeah, ETA right now is really off, especially in urban areas. OM suggests very shorter ETAs compared to real life. Regarding routes, I usually don't have problems, considering OM don't have access to traffic data. It suggest quite good routes, trying to use the high priority routes etc.

**Comment by biodranik** [comment]
Did you read #1845 ?

@Tmpod it would be great if you can properly penalize traffic lights and turns in the existing implementation ;-)

CC @vng

**Comment by Explosion-Scratch** [comment]
Can you give an example route for somewhere not nearby you @Gravity2106
